app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -83,12 +83,15 @@ class ThreadFragment : Fragment() {
     @Inject
     lateinit var localSettings: LocalSettings
 
+    @Inject
+    lateinit var injectedThreadAdapter: ThreadAdapter
```

Comment:
Why not just threadAdapter ?

Reply:
Because there is already a `threadAdapter` here.